src/main/resources/templates/db_scripts/migration/extract_credits_from_expenditures.sql

```diff
@@ -0,0 +1,59 @@
+-- Extract credits from expenditures in each budget
+-- 1. Calculate all credits values within budget fiscal year
+-- 2. Set to budget credits fields
+-- 3. Subtract from expenditures amount
+
+WITH credit_sums AS (
+    SELECT
+        budget.id AS budget_id,
+        SUM((trx.jsonb->>'amount')::numeric) AS total_credits
+    FROM ${myuniversity}_${mymodule}.budget AS budget
+    JOIN ${myuniversity}_${mymodule}.transaction AS trx
+        ON trx.jsonb->>'fiscalYearId' = budget.jsonb->>'fiscalYearId' AND
+		trx.jsonb->>'fromFundId' = budget.jsonb->>'fundId'
```

Comment:
please check this statement:		trx.jsonb->>'fromFundId'
Credit transactions have toFundId populated. FromFundId is empty for them
![image](https://github.com/user-attachments/assets/2fe66a4e-53f0-4a77-a04c-09af9dd6fedc)

Reply:
Good catch thanks, updated